scss/_custom-forms.scss

```diff
@@ -94,7 +94,7 @@
 
 .custom-radio {
   .custom-control-indicator {
-    border-radius: $custom-radio-indicator-border-radius;
+    @include border-radius($custom-radio-indicator-border-radius);
```

Comment:
Please remove it from here, if $enable-rounded is set to false we will lose the styles for the radio